source/administration/changelog.md

```diff
@@ -4,6 +4,129 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.0
+
+Release date: 2018-06-16
+
+### Highlights
+
+#### Plugin Intercept
+ - Adds support for plugins to intercept posts prior to saving them into the database.
+ - Supports use cases such as auto-detecting and censoring restricted words, and auto-linking phrases. [Read our forum post to learn more](https://forum.mattermost.org/t/coming-soon-apiv4-mattermost-post-intercept/4982).
+ 
+#### Permissions Schemes
+ - System Scheme now sets the default permissions inherited system wide by System Admins, Team Admins, Channel Admins and everyone else. 
+ - Added new Team Schemes to override the default permissions in specific teams for Team Admins, Channel Admins and all other team members.
+
+#### Increased Character Limit on Posts
+ - Increased character limit to 16,000 on new deployments to allow posting long messages and to allow better Markdown formatting, including tables.
+ - For existing deployments, read [how to migrate your system](https://docs.mattermost.com/administration/important-upgrade-notes.html) to support the increased character limit.
+ 
+#### Combined Join/Leave Messages
+ - System messages related to joining, leaving, adding and removing people from channels are combined into a single message, allowing these messages to not take excess space in channels.
```

Comment:
into a single message, allowing these messages to not take excess space in channels > into a single message to save space in channels.